src/Analyzers/Reliability/InvokableControllersAnalyzer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace JumpTwentyFour\ProjectAnalyzers\Analyzers\Reliability;
+
+use Enlightn\Enlightn\Analyzers\Reliability\ReliabilityAnalyzer;
+use Illuminate\Filesystem\Filesystem;
+use JumpTwentyFour\ProjectAnalyzers\Concerns\AnalyzesFiles;
+
+class InvokableControllersAnalyzer extends ReliabilityAnalyzer
+{
+    use AnalyzesFiles;
+
+    /**
+     * The title describing the analyzer.
+     *
+     * @var string|null
+     */
+    public $title = 'Analyzes whether all controllers are single action controllers.';
```

Comment:
Shall we just add the word `invokable` here as well to then match the error message in terms of meaning?

Reply:
Updated